sdk/include/reactos/undocshell.h

```diff
@@ -93,6 +93,9 @@ HRESULT WINAPI SHILCreateFromPathW (
     LPITEMIDLIST * ppidl,
     DWORD *attributes);
 
+HRESULT WINAPI SHInvokeCommand(
```

Comment:
Could you move also add the other new undocumented functions here to `sdk/include/reactos/shlwapi_undoc.h` as well?

Reply:
The other functions hidden under version 6+ are not that useful since we can't call them from shell32.